index.html

```diff
@@ -0,0 +1,214 @@
+            university.
+          </p>
+          <h2 class="hero-date">2022.04.12(TUE)~14(THU)</h2>
+          <span class="hero-address"
+            >@ Twins Building, Arba Minch University</span
+          >
+        </div>
+      </section>
+      <section id="the-training">
+        <div class="training-section desktop-padding">
+          <h2 class="the-training-title the-underline">The Training</h2>
+          <div class="events">
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/lecture-talk.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Lecture</h2>
+              <p>Attend the lectures of the web development course training.</p>
+            </div>
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/class-project.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Projects</h2>
+              <p>
+                Finish class projects to train yourself what you learned in the
+                lecture of the course.
+              </p>
+            </div>
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/workshop.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Workshop</h2>
+              <p>
+                Practice web development techniques in our workshop session.
+              </p>
+            </div>
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/lecture.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Forum</h2>
+              <p>
+                Have the time to share your opinions and your thoughts with
+                expert developers.
+              </p>
+            </div>
+            <div class="event-item">
+              <img
+                class="event-image"
+                src="./images/icons/certification.svg"
+                alt="Lecture Talk"
+              />
+              <h2>Certification</h2>
+              <p>
+                Get a web development certificate verfied by Arba Minch
+                University.
+              </p>
+            </div>
+          </div>
+          <button type="button" class="register-btn">
+            Register For The Training
+          </button>
+          <a href="#" class="see-all-program">See the Whole Program</a>
+        </div>
+      </section>
+      <section id="trainers">
+        <div class="desktop-padding">
+          <h2 class="trainers-section-title the-underline">Trainers</h2>
+          <div id="trainers-container"></div>
+          <button id="more-trainers" class="more-trainers">
+            More <img src="images/icons/down.svg" alt="" />
+          </button>
+        </div>
+      </section>
```

Comment:
- [ ] According to the design specifications, the speakers(trainers in your case) section should have six different cards. At the moment, yours is only five. Please include one more trainer so that it matches the design.

### Your Design
![image](https://user-images.githubusercontent.com/43995830/157901150-c2cf7f53-6aab-4481-94cc-9b202f5ce429.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/157901265-926a21e4-0f27-4772-b81a-981f3ca00819.png)